### PR TITLE
Upgrade

### PR DESCRIPTION
**Pull request by andrewsource147**

### CODE REVIEW

File: ts-client/src/vault/tests/affiliate.test.ts [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
 
 import VaultImpl from '..';
 import { airDropSol } from './utils';
 import { NATIVE_MINT } from '@solana/spl-token';
+import fs from "fs";
+import os from 'os'
+
+let mockWallet = new Wallet(new Keypair());
+
+// function loadKeypairFromFile(filename: string): Keypair {
```
Comment (codewithgun) [inline ts-client/src/vault/tests/affiliate.test.ts:12]: Can you remove these?

File: ts-client/src/vault/tests/affiliate.test.ts [hunk 2]
```diff
@@ -26,17 +38,9 @@ describe('Interact with Vault in devnet', () => {
     });
   });
 
-  test('Test affiliate init user, check balance, deposits, then withdraw all', async () => {
+  test.only('Test affiliate init user, check balance, deposits, then withdraw all', async () => {
```
Comment (codewithgun) [inline ts-client/src/vault/tests/affiliate.test.ts:41]: Can you remove `.only`?

**Review by codewithgun — COMMENTED** [review]
I'm not sure whether it's good to maintain 2 different package manager lock files or not. Maybe we should just use one?

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+### Deprecated
+
+### Removed
+
+### Fixed
+
+### Security
+
+## @mercurial-finance/vault-sdk [2.0.0] - PR #139
```
Comment (codewithgun) [inline CHANGELOG.md:22]: PR #124

File: ts-client/src/vault/tests/affiliate.test.ts (hunk 1 above)
Comment (andrewsource147) [reply ts-client/src/vault/tests/affiliate.test.ts:12]: fixed

File: ts-client/src/vault/tests/affiliate.test.ts (hunk 2 above)
Comment (andrewsource147) [reply ts-client/src/vault/tests/affiliate.test.ts:41]: fixed

File: CHANGELOG.md (hunk 3 above)
Comment (andrewsource147) [reply CHANGELOG.md]: fixed

**Review by codewithgun — APPROVED** [review]

**Pull request merged**